tests/clients/test_securitydata.py

```diff
@@ -1384,3 +1384,33 @@ def test_search_all_file_events_calls_search_with_expected_params_when_pg_token_
             data='{"groupClause":"AND", "groups":[], "pgToken":"abc", "pgSize":10000}',
         )
         assert response is successful_response
+
+    @pytest.mark.parametrize(
```

Comment:
Why parametrize the test if there is only a single test case?

Reply:
There are two cases, we want to add an escape to a string that isn't already escaped: `('1234_"abcde"', r'1234_\"abcde\"')` and leave untouched if the string is already escaped: `(r'1234_\"abcde\"', r'1234_\"abcde\"')`.

Reply:
Oh whoops, I see. That might be more readable as a separate tests named `test_search_all_file_events_does_not_escape_already_escaped_quotes()`

Reply:
That's true, I'll make it two tests.